beamer/deploy/commands.py

```diff
@@ -95,3 +96,101 @@ def deploy_base(
     path = artifacts_dir / "base.deployment.json"
     beamer.deploy.artifacts.generate(path, account.address, (resolver,))
     log.info("Generated artifact", path=str(path))
+
+
+@click.command("deploy")
+@click.option(
+    "--rpc-file",
+    "rpc_file",
+    type=click.Path(exists=True, file_okay=True, dir_okay=False, path_type=Path),
+    required=True,
+    metavar="PATH",
+    help="Path to the RPC config file.",
+)
+@click.option(
+    "--keystore-file",
+    type=click.Path(exists=True, file_okay=True, dir_okay=False, path_type=Path),
+    required=True,
+    metavar="PATH",
+    help="Path to the keystore file.",
+)
+@click.password_option(
+    "--password",
+    type=str,
+    default="",
+    prompt=False,
+    help="The password needed to unlock the keystore file.",
+)
+@click.option(
+    "--artifacts-dir",
+    type=click.Path(exists=True, file_okay=False, dir_okay=True, path_type=Path),
+    required=True,
+    metavar="DIR",
+    help="The directory to store contract deployment artifacts in.",
+)
+@click.option(
+    "--deploy-mintable-token",
+    is_flag=True,
+    help="Deploy MintableToken.sol on the specified chains.",
+)
+@click.argument(
+    "chain.json",
+    nargs=-1,
+    required=True,
+    type=click.Path(exists=True, file_okay=True, dir_okay=False, path_type=Path),
+)
+def deploy(
+    rpc_file: Path,
+    keystore_file: Path,
+    password: str,
+    artifacts_dir: Path,
+    deploy_mintable_token: bool,
+    **chains: dict,
+) -> None:
+    """Deploy L2 Beamer contracts on the specified chains and set up the trusted call chain."""
+    beamer.util.setup_logging(log_level="DEBUG", log_json=False)
+    _ensure_commit_is_on_remote()
+
+    rpc_info = beamer.deploy.config.load_rpc_info(rpc_file)
+    log.info("Loaded RPC file")
+
+    account = beamer.util.account_from_keyfile(keystore_file, password)
+    log.info("Loaded keyfile", address=account.address)
```

Comment:
nitpick: We wanted to get rid of the notion `keyfile` and instead use `keystore (file)`. I just saw that we still have this notion in the tests, but would be nice to remove it

Reply:
I just saw that we also still have named the account_from_keyfile function like that

Reply:
ideally, yes, we should get completely get rid of the naming `keyfile`. 
At least for the interface to the user (options and logs) we were already consistent, but ideally yes, we completely get rid of `keyfile`

Reply:
I have update naming in the log output, but I'd rather leave the whole-repo keyfile -> keystore file conversion for a separate PR, if that's fine with you.

Reply:
Yes, I would do the same.